### DISCUSSION TITLE
Codegen plugin typescript-urql-graphcache, what's this about?

### DISCUSSION THREAD

**Discussion by frederikhors** [urql-graphql/urql#1903] [Q&A] [answered]
I just found https://github.com/dotansimha/graphql-code-generator/tree/master/packages/plugins/typescript/urql-graphcache.

There is no Readme.

What's this about?

**Comment by JoviDeCroock** [comment, marked as answer]
it generates a config for graphcache so updaters,... are strictly typed so the output of that plugin can be used as a generic to cacheExchange

**Reply by frederikhors** [reply]
Are we going to write a Readme and a page here: https://www.graphql-code-generator.com/docs/plugins/index?

**Reply by frederikhors** [reply]
Anyway i'm using it and it works amazing!
